kili/utils/bucket.py

```diff
@@ -25,12 +26,16 @@ def request_signed_urls(auth: KiliAuth, project_id: str, size: int):
         project_id: the project Id
         size: the amount of upload signed URL to query
     """
-    payload = {
-        "projectID": project_id,
-        "size": size,
-    }
-    urls_response = auth.client.execute(GQL_CREATE_UPLOAD_BUCKET_SIGNED_URLS, payload)
-    return urls_response["data"]["urls"]
+    signed_urls = []
+    while len(signed_urls) < size:
+        nb_url_requested = min(MAX_NUMBER_SIGNED_URLS_TO_FETCH, size - len(signed_urls))
```

Comment:
I think we should let it crash. This is the caller of `request_signed_urls` that should make sure there not too many rsl requested.